libs/model/src/models/address.ts

```diff
@@ -91,15 +91,15 @@ export default (
       },
       ghost_address: {
         type: dataTypes.BOOLEAN,
-        allowNull: false,
+        allowNull: true,
```

Comment:
Let's remove `defaultValue` here to match.

Reply:
removed... but adding default values and sequences to next reconciliation phase